Engine/src/Engine/Components.h

```diff
@@ -58,6 +58,21 @@ namespace Engine
 		GLuint texID;
 	};
 
+	//A component containing animation data
+	struct AnimationComponent
```

Comment:
Make sure to add this into deserialization Serializer.cpp.
LMK if I can help with anything